.eslintrc.js

```diff
@@ -95,10 +98,11 @@ module.exports = {
         // valid case of class method overloads in typescript
         'no-dupe-class-members': 'off',
         '@typescript-eslint/ban-ts-comment': 'off',
-        //  Missing return type on function 
+        //  Missing return type on function
         '@typescript-eslint/explicit-module-boundary-types': 'off',
         // note you must disable the base rule as it can report incorrect errors
-        "no-use-before-define": "off",
-        "@typescript-eslint/no-use-before-define": ["error"]
+        'no-use-before-define': 'off',
+        '@typescript-eslint/no-use-before-define': ['error'],
+        'require-await': ['error'],
```

Comment:
apart from this line, this file was only linted. I can remove these linting changes if you want.